prisma/schema.prisma

```diff
@@ -334,4 +334,12 @@ model Landing_Page {
   content         String?
   user            User @relation(fields: [userId], references: [id])
   userId          Int
+}
+
+model Feedback {
+  id                Int      @id @default(autoincrement())
+  datetime_recorded DateTime?
+  rating            Int?
```

Comment:
Shouldn't these be required, based on what the API sends?

Reply:
fixed